tutorial/lesson_07_multi_stage_pipelines.cpp

```diff
@@ -30,7 +31,7 @@ int main(int argc, char **argv) {
     // first horizontally, and then vertically.
     {
         // Take a color 8-bit input
-        Image<uint8_t> input = load<uint8_t>("images/rgb.png");
+        Image<uint8_t> input = load<Image<uint8_t>>("images/rgb.png");
```

Comment:
OK, but then how do we handle the save() calls?

     bool save();  // false == failue
     void save();   // aborts or throws 

....not gonna work.